src/templates/podcast-template.js

```diff
@@ -124,41 +125,9 @@ const Podcast = ({ pageContext }) => {
           </div>
         </div>
       </section>
-
-      <div
-        style={{
-          display: 'flex',
-          justifyContent: 'space-around',
-          alignItems: 'center',
-          margin: '0 auto',
-        }}
-      >
-        <nav aria-label="Page navigation">
-          <ul className="pagination">
-            {!isFirstPage && (
-              <li>
-                <Link to={prevPage} rel="prev">
-                  {/* <span aria-hidden="true">&laquo;</span> */}
-                  <span aria-hidden="true">Previous</span>
-                </Link>
-              </li>
-            )}
-            {Array.from({ length: totalPages }, (_, index) => (
-              <li key={index}>
-                <Link to={`/podcast/${index === 0 ? '' : index + 1}`}>{index + 1}</Link>
-              </li>
-            ))}
-            {!isLastPage && (
-              <li>
-                <Link to={nextPage} rel="next">
-                  {/* <span aria-hidden="true">&raquo;</span> */}
-                  <span aria-hidden="true">Next</span>
-                </Link>
-              </li>
-            )}
-          </ul>
-        </nav>
-      </div>
+      <nav aria-label="Page navigation">
+        <Pagination pageContext={pageContext} type={'route'} />
```

Comment:
Please change `type={'route'}` to `type="route"`

Reply:
done